language-support/java/codegen/src/test/java/com/digitalasset/testing/GenMapTestFor1_11AndFor1_12ndFor1_13AndFor1_14AndFor1_15AndFor1_dev.java

```diff
@@ -69,6 +69,21 @@ void toValuePreservesOrder() {
     assertEquals(keys[2], pair3());
   }
 
+  @Test
```

Comment:
I think we need a lot more tests here, a few things of particular interest:
* type parameters
* recursive types
* cross-module references

Reply:
seems fine